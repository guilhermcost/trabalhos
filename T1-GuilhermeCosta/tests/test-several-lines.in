// linha 1 
// outra linha
void main(void) // linha 3
{
    int aTab;   // linha 5 com tab
}               // linha 6

