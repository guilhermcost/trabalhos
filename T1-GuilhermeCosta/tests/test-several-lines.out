(3,KEY,"void")
(3,ID,"main")
(3,SYM,"(")
(3,KEY,"void")
(3,SYM,")")
(4,SYM,"{")
(5,KEY,"int")
(5,ID,"aTab")
(5,SYM,";")
(6,SYM,"}")
